src/benchmarks/beluga_vs_nav2_multi_dataset/reports/nominal_report/sections/main_corpus.inc

```diff
@@ -0,0 +1,245 @@
+
+.. repl-quiet::
+
+    def get_bagfile_info(bagpath):
```

Comment:
@glpuga fyi, feel free to ignore: there's also [`rosbag2_py.Info().read_metadata(bag_path)`](https://github.com/ros2/rosbag2/blob/786c3c4b8ab05271630371cf515130ba02a9cde8/rosbag2_py/rosbag2_py/_info.pyi#L9) for this too.

Reply:
Lets keep it in mind for the next time, but since this has no other testing than surving running report generation with the playground in a bunch of different error states, let's keep it like this. I'll add a TODO so that we update this when we update  a do a new measurement campaign.